python/pyspark/sql/dataframe.py

```diff
@@ -1303,9 +1312,11 @@ def fillna(self, value, subset=None):
         +---+------+-------+
         """
         if not isinstance(value, (float, int, long, basestring, dict)):
```

Comment:
I omitted it just because it wasn't failing for this if, but indeed, I'm a bit more on the side of putting it in even if just for completeness. Makes reading the code much saner if we have the if for bool